src/test/java/com/google/api/generator/engine/JavaCodeGeneratorTest.java

```diff
@@ -63,8 +63,9 @@
 import org.junit.Test;
 
 public class JavaCodeGeneratorTest {
+
   private static final String GOLDENFILES_DIRECTORY =
-      "src/test/java/com/google/api/generator/engine/goldens/";
+      "../src/test/java/com/google/api/generator/engine/goldens/";
```

Comment:
Would it be possible to change the working directory for surefire or failsafe instead?

Reply:
Good idea. I'll give it a try.

Reply:
This suggestion gets bonus points for both simplifying the PR *and* fixing the bazel build.